frontend/integration-tests/protractor.conf.ts

```diff
@@ -87,6 +89,7 @@ export const config: Config = {
     return new Promise(resolve => htmlReporter.afterLaunch(resolve.bind(this, exitCode)));
   },
   suites: {
+    ...pluginSuites,
```

Comment:
AFAIK, In this case the spread operator [1] is equivalent to lodash's [2] `defaults` utility, in both cases we override the 
`pluginSuites` map with default values from the core test suites.

For example using the spread operator, the plugins `foo` test is shadowed by the default core `foo` test. 
```
var pluginSuites = { foo: 'bar' };

var mergedObj = { ...pluginSuites, foo: 'baz'  };
// Object { foo: "baz" }
```

this is equivalent to the behaviour of `_.defaults`:
```
var pluginSuites = { foo: 'bar' };

var mergedObj = _.defaults(
  {
    // core test suites, to be moved to console-app later on
    foo: 'baz',
  },
  pluginSuites,
)
// Object { foo: "baz" }
```

IMHO, it's better to use the language spread operator than an imported utility in this case, WDYT ? 

[1] https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax#Spread_in_object_literals
[2] https://lodash.com/docs/4.17.14#defaults

Reply:
> AFAIK, In this case the spread operator [1] is equivalent to lodash's [2] `defaults` utility

Correct, it is equivalent in terms of behavior / effect.

I just find the style of writing "core (base) first, plugins (defaults) second" more correct, instead of "plugins (base) first, core (overrides) second". That's the only reason for this suggestion :smile: 

Using an external library (Lodash) shouldn't really matter here; this code is executed in Node.js env. and not subject to optimization, beyond the minimum necessary TS-to-JS transpilation done via `ts-node`.